src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands;
+
+public abstract class ExportCommand extends CommandWithFile {
+    public static final String COMMAND_WORD = "export";
+
+    public static final String PROPERTIES = "properties";
+
+    public static final String BUYERS = "buyers";
```

Comment:
Should there be a central source of the words "property", "buyer", "properties", "buyers"?

Reply:
Should have some sources here and there. It's ok to repeat words. 
Will consider moving Properties and Buyers in the next PR to a more general Command.